LICENSES/vendor/github.com/benbjohnson/clock/LICENSE

```diff
@@ -0,0 +1,25 @@
+= vendor/github.com/benbjohnson/clock licensed under: =
```

Comment:
I'm sad to see test-only deps bleeding out of these libraries and impacting consumers... can we open an upstream issue for them to clean this up?